Document robots.txt AI agent access check (#5330)
mintlify/docs: guides/geo.mdx

You can view your LLMs.txt by appending `/llms.txt` to your documentation URL.

### Allow AI agents in robots.txt

> Your `robots.txt` file controls which bots can crawl your site. If it blocks AI user agents, tools like ChatGPT, Claude, and Perplexity cannot read your documentation and will not cite it in answers.

Reviewer: ```suggestion
Your `robots.txt` file controls which bots can crawl your site. If it blocks AI user agents, tools like ChatGPT, Claude, and Perplexity cannot read your documentation nor cite it in answers.
```

Mintlify's auto-generated `robots.txt` allows all crawlers by default. If you use a [custom robots.txt file](/optimize/seo#custom-sitemaps-and-robotstxt-files), make sure it does not block AI agents. The most common AI user agents include:

- `GPTBot`, `OAI-SearchBot`, `ChatGPT-User` (OpenAI)
- `ClaudeBot`, `Claude-User` (Anthropic)
- `PerplexityBot` (Perplexity)
- `Google-Extended` (Gemini)

> A robots.txt that blocks all crawlers also blocks AI agents:

Reviewer: ```suggestion
A `robots.txt` that blocks all crawlers also blocks AI agents:
```
